.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```

Comment:
What is the purpose of titles, labels and assignees if they are to be kept blank>

Reply:
It was intended for a collaborator to assign people, create labels on issue creation. if you want i can remove as not many would be using it.

Reply:
Yeah we should remove them. We don't expect issue creators to assign labels or assignees.

Reply:
@abhigyank done.